### DISCUSSION TITLE
Damacana su sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#293] [Zeka Sorusu]
Elde 31, 19 ve 13 litrelik üç damacana var. Bunlardan 31 litrelik olanı boş diğerleri suyla dolu. Bu üç kabı kullanarak bu 32 litre suyu 2 eşit kısma nasıl ayırabilirsiniz?

**Comment by AhmetEmirErsoz** [comment]
31 litrelik damacananın 16 litrelik kısmı dolabilie çünkü
19 ve 13 litrelik damacanaları 2 değil 1 damacana olarak sayıyoruz ve 32 sayısını 2 ye bölüyoruz böyle 32 litreliğin yarısı 31 litreliğinde 16 litresi dolmuş oluyor
